### DISCUSSION TITLE
Community public speaking practice 2021-05-07

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#1532] [Show and tell]
Notes from @kaiwalyakoparkar 

> We are planning to conduct a simple public speaking session for all (This is not conducted by Eddie or anybody in general but by self will of some community members). We are planning to conduct it on (07/ 05/ 2021) 3:00pm to 4:00 pm IST.
please react to this message by :thumbsup: or :thumbsdown: so that we can understand your availability and change the schedule accordingly (If needed). We want more people to participate so kindly give it a try :smile: .

> It's all about learning so don't hesitate to participate it's very begginer friendly and ..  
Fun fact: we all beginners who conducts the session :grin::thumbsup:

**Rules:**
- mod must be present to make sure everyone is inclusive and the CoC is enforced (everyone please read [Code of Conduct](http://coc.eddiehub.org) first)
- be patient and supportive to everyone on the call

**To join in:**
- start a new thread below with your talk and description
- when some gives their talk you can reply to their thread to leave feedback (remember to include good feedback too)

**Comment by kaiwalyakoparkar** [comment]
My speaking topic: **What EddieHub has given me so far, And how you can take maximum of EddieHub**
I would love to get you suggestion about my speaking below (By commenting down) See you at the talk

**Reply by verma-kunal** [reply]
This will be an interesting topic for all the new folks !
I myself being relatively new to EddieHub, would learn a lot about the community !!😃

**Reply by imvisp** [reply]
yes it would be good starting point topic :+1:

**Reply by kaiwalyakoparkar** [reply]
> This will be an interesting topic for all the new folks !
> I myself being relatively new to EddieHub, would learn a lot about the community !!

Good to know that. See you there.

**Reply by Evavic44** [reply]
This is a great idea, not just for new members but also for old members to appreciate Eddiehub community. 😎

**Comment by imvisp** [comment]
I think it would be grate if we do this exercise daily, by doing this it becomes habit for new folks.

**Reply by kaiwalyakoparkar** [reply]
What is your topic btw.

**Reply by imvisp** [reply]
i want some clear idea about python
- what after basics and all
- how to stay productive

**Comment by felixfaisal** [comment]
My speaking topic: **Why Rust should replace existing C/C++ codebase** 
I wont be diving into anything advanced technical just pros and cons, Do let me know if this would be something everyone is interested in

**Reply by kaiwalyakoparkar** [reply]
Good to hear that

**Reply by imvisp** [reply]
see you there :+1:

**Comment by kaiwalyakoparkar** [comment]
Key Points in the session :star:

**Comment by Vyvy-vi** [comment]
### **Thread for Summary of Community Public speaking call - 2021-05-07**
Enter the details from this session, that you spoke in the call and about your experience as comments under this message.

Thanks for attending the call! 🎉 🎉

**Reply by verma-kunal** [reply]
Although I didn't speak on a particular topic today, I really enjoyed learning about other folks's experiences with EddieHub and communities in general.
Also, I tried to interact and involve a bit in the discussion as well.
Looking forward for more such sessions and talks !!😃

**Reply by imvisp** [reply]
yeah same :+1:

**Reply by imvisp** [reply]
really happy that our idea works. :100:

**Reply by Yuvrajhere** [reply]
It was my first time speaking in EddieHub server, really felt nice to get the opportunity. Thanks to everyone for listening. I just spoke about myself and my experience in EddieHub so far. 😋

**Reply by Rahma-AlWadhahi** [reply]
I like how people are welcoming and helping each other no matter where they are in their journey. I learn a lot every day from everyone even when I don’t know about a certain topic and I try to search for information and give them hints to how might they solve their issues.
In addition, speaking and describing your issue loud sometimes helps you and other people when trying to solve it together. [Rubber Duck-ing](https://medium.com/@katiebrouwers/why-rubber-ducking-is-one-of-your-greatest-resources-as-a-developer-99ac0ee5b70a)

**Reply by Evavic44** [reply]
I talked about how EddieHub's slogan `Collaboration first, Code second` drew me to the community, because a lot of communities raises the bar for collaborating and contributing so high that entry level or new developers like myself can't be able to join or collaborate effectively. Which leads me to an idea I'll be sharing in the suggestion thread. 😎

**Reply by Evavic44** [reply]
> I like how people are welcoming and helping each other no matter where they are in their journey. I learn a lot every day from everyone even when I don’t know about a certain topic and I try to search for information and give them hints to how might they solve their issues.
> In addition, speaking and describing your issue loud sometimes helps you and other people when trying to solve it together. [Rubber Duck-ing](https://medium.com/@katiebrouwers/why-rubber-ducking-is-one-of-your-greatest-resources-as-a-developer-99ac0ee5b70a)

You are great @Rahma-AlWadhahi 🆒 💯 😇

**Reply by eddiejaoude** [reply]
Wow amazing to read all of this ❤️ , you are all awesome! 💥

**Comment by Vyvy-vi** [comment]
### **Thread for Suggestions for next call**
Comment all the wonderful suggestions you have for the community call, with respect to timing, dates, topics, how to conduct, how to invite people etc. below.
This would help us narrow the field of conversation to one thread, and thus interpret suggestions more easily and effectively.

Thanks for all the suggestions! 🎉 🎉

**Reply by kaiwalyakoparkar** [reply]
Polling to match most time zones

**Reply by imvisp** [reply]
maybe it's ok to this time i am fine with this :+1:

**Reply by verma-kunal** [reply]
Talking about suggestion, in the future calls we can all decide on a particular topic to talk about and other folks can add their experiences to that !
Also, I think that keeping the session regular i.e weekly, would really become a part of the community and more new folks would be able to join in !!🙌

**Reply by felixfaisal** [reply]
**Retrospective for the week for everyone**
Where a person can talk about how their week went and then can categorize it into 
Green - Went well 
Orange - Went sorta well, sorta bad 
Red - Went Bad 
This can help us improve and remove the red and orange in the future, While also being able to appreciate all our greens. Also talking about this, you can get help from other community members or you can also offer your help.

**Reply by imvisp** [reply]
> Talking about suggestion, in the future calls we can all decide on a particular topic to talk about and other folks can add their experiences to that !
> Also, I think that keeping the session regular i.e weekly, would really become a part of the community and more new folks would be able to join in !!

Nice idea :+1:

**Reply by Yuvrajhere** [reply]
My suggestions :
- Having a topic of discussion should improve the quality as we will already know what we going to learn and speak about.
- Giving few minutes in start for new comers to introduce themselves in short.
- Keeping few minutes at end of session for feedbacks, specially from new comers.
- in the middle of this, the whole discussion should be around the planned topic.

**Reply by Rahma-AlWadhahi** [reply]
> Polling to match most time zones

I think as a start we can add a poll, then we might want to add a calendar invite for that session. For example,

**Public Speaking Practice - < topic to be discussed > - date and time with timezone**

**Reply by kaiwalyakoparkar** [reply]
All the ideas are super great. We need to think about this!

**Reply by Evavic44** [reply]
Topic ideas for the next community radio session.
 
- [ ] `How did you get into Tech.` This could spark up a good conversation that new members can actively participate in. Also might be a fun topic to learn a thing or two from.

- [ ] At what level do you get to before you can confidently add professional titles to your profile? Like `Software Engineer`

**Reply by kaiwalyakoparkar** [reply]
@Evavic44  I think the first one is great. The second one is also great but we will need all the experience people in our community to actually answer that. 

We can take 1st into consideration coz everyone has a story of them getting into tech

**Reply by Evavic44** [reply]
> @Evavic44 I think the first one is great. The second one is also great but we will need all the experience people in our community to actually answer that.
> 
> We can take 1st into consideration coz everyone has a story of them getting into tech

I agree. I raised the second one if experienced devs would be joining the session, like Eddie did last time.

**Reply by kaiwalyakoparkar** [reply]
Yup! We can conduct this in the Community call (Which is conducted last Saturday of every month)

**Reply by imvisp** [reply]
> @Evavic44 I think the first one is great. The second one is also great but we will need all the experience people in our community to actually answer that.
> 
> We can take 1st into consideration coz everyone has a story of them getting into tech

yes, the first one really good idea. :+1:

**Comment by kaiwalyakoparkar** [comment]
:heart:  Saturday Public Speaking session **15/ 05/ 2021 6:00 PM IST**  :heart: 

:point_down:  Feel free to add your ***suggestions*** for improvement of this initiative and ***idea*** for next session :point_down: 

Also share about your experience on socials (And don't forget to tag your peers :wink: ) and how it is helping you!

**Reply by ghost** [reply]
Amazing Session 👍🏻 ❤️. My Rating infinity/five 🥇. Thanks to **Hubber Spirit** for motivating me

**Reply by kaiwalyakoparkar** [reply]
Hope to hear you in many more sessions :smile: :tada:

**Reply by ayushtamra** [reply]
The session was awesome..!! Thanks for such an amazing session❤️.
Rating ♾/5, no doubt

**Reply by Evavic44** [reply]
It was awesome and cool, I feel my confidence for public speaking is greatly improving, all thanks to everyone at the community for making this happen. 😇 
Can't wait to join in the next session.

**Comment by kaiwalyakoparkar** [comment]
:heart:  Saturday Public Speaking session **22/ 05/ 2021 6:00 PM IST** :heart:

:point_down: Feel free to add your suggestions for improvement of this initiative and idea for next session :point_down:

Also share about your experience on socials (And don't forget to tag your peers :wink: ) and how it is helping you!

**Reply by Akshu-on-github** [reply]
My two cents:
- *Documentation*: Initially, I thought of documentation as a big wall of text. But then I worked through my inhibitions and recommend it ever since!
- *Organized Headings*: We see heading tags from H1 through H6 in Google docs, markdown, and .html files. Organising those, provides a clear structure that's a godsend for screen readers and auto-generated indexes. And hey, +1 for accessiblity
- *Clean Code*: A popular point - it was brought up multiple times in today's call! But yes, use sensible variable names, write sensible comments and use sensible logic!

**Reply by lainq** [reply]
I couldn't speak today(as my mic is on leave(It's actually broken :expressionless: )). But, I would like to share the biggest mistake that I made.

- Assuming that you know stuffs. When I started out, I assumed that I knew stuffs that I actually didn't know. 

- Jumping between topics. I used to jump between technologies without learning anything completely. 

I have learned from these mistakes and I am trying to improve my skills.

Thank you for reading 🙂

**Comment by iason9** [comment]
My speaking topic: "How to stay motivated and keep coding and creating new projects" 🥇 
➡️ why not talk about methods and ways to stay motivated in hard times when you feel really down. 
Few points we could talk about : 
- The Pomodoro Technique
- Starting small step by step not going all the way to create big projects.
- get out as soon as possible the MVP (Minimum viable product).
- take time to relax and get back stronger and more motivated.

**Reply by kaiwalyakoparkar** [reply]
That's a great topic! Could you please tell the points on which we can motivate the newcomers to speak if they are not that experienced and exposed to projects and coding?

**Reply by ghost** [reply]
?? what do we have in the next public speaking I am not understanding anything 😵

**Reply by starlightknown** [reply]
Thanks @iason9 I can definitely use this talk, staying on track is really important, one thing I noticed that motivation is great, someone gives a talk a day and you're like "yass let's do it"  later someday you lose it and you need constant pushes which doesn't help much. I would like to hear a way out from someone

**Reply by iason9** [reply]
> That's a great topic! Could you please tell the points on which we can motivate the newcomers to speak if they are not that experienced and exposed to projects and coding?

I updated the suggestion..

**Reply by iason9** [reply]
> Thanks @iason9 I can definitely use this talk, staying on track is really important, one thing I noticed that motivation is great, someone gives a talk a day and you're like "yass let's do it" later someday you lose it and you need constant pushes which doesn't help much. I would like to hear a way out from someone

I am sure with the experience of each one, we could find great ways..

**Comment by verma-kunal** [comment]
i have a topic idea for the upcoming Public Speaking session, maybe we can have it as 
**"Best health practices to follow as a Developer"**
Feedbacks & suggestions are welcomed 😇

**Reply by Akshu-on-github** [reply]
I'd love to hear more on this!

**Reply by ashwinexe** [reply]
This would help all of us, we spent good time on screens.

**Reply by kaiwalyakoparkar** [reply]
Will the beginners be able to speak on it. Our main aim is to make them feel to speak. Personally, I won't be able to speak more on this topic! :sweat_smile:

**Reply by verma-kunal** [reply]
Hmm🤔
I think beginners can like share the problems they face by sitting long hours infront of the screen, & we'll also be joined by folks with some experiences as well, so we can all guide each other jn that aspect !!
Let's see how the community goes with the topic 😇

**Reply by kaiwalyakoparkar** [reply]
Umm. In that case, most of the session will get focused on listening which we want to avoid :)

**Reply by verma-kunal** [reply]
Hmm, yeah agreed to that! Maybe we can then discuss this in our next community call & have @nhcarrigan and @eddiejaoude share some insights on this too !!

**Comment by kaiwalyakoparkar** [comment]
Topic suggestion for 5 Jun
"Discuss about your college/school experiences"

**Reply by viki4545** [reply]
yes i would like to listen experiences and enjoy the community session

**Reply by verma-kunal** [reply]
This is an interesting topic!! Folks would have a lot to speak on this according to me & we'll have a fun session 😃

**Reply by kaiwalyakoparkar** [reply]
> This is an interesting topic!! Folks would have a lot to speak on this according to me & we'll have a fun session

Exactly!

**Reply by archismansaha** [reply]
It would be nice to listen everyone's experience 🔥

**Comment by kaiwalyakoparkar** [comment]
:heart:  Saturday Public Speaking session **5/ 06/ 2021 6:00 PM IST** :heart:

:point_down: Feel free to add your suggestions for improvement of this initiative and idea for next session :point_down:

Also share about your experience on socials (And don't forget to tag your peers :wink: ) and how it is helping you!

**Comment by verma-kunal** [comment]
Hey everyone 👋
Due to un-avoidable circumstances & some important work of some community members, we would not be able to conduct our public speaking session today (as there is no host available to conduct the session) !!
If anyone from the community is up for hosting this session today, we all would be really happy to support! But if not, unfortunately we wouldn't be able to meet 😕

We can either have a session tomorrow ( Sunday ) at 6pm IST or the next Saturday itself, whichever the community decides to do !!

Let's Vote here:
- Sunday ( 👀   )
- Next Saturday ( 🚀  )

**Reply by Vyvy-vi** [reply]
I'll be up for moderating the call.
I could host it too, if anyone's interested in attending

**Reply by ghost** [reply]
Umm! Maybe now we can have the call later? Cause many people now might think that today's call is cancelled, I don't think if we make a announcement know, many people will be able to join on an short notice 🤔 😟

**Comment by ItsMeIshan** [comment]
My suggestion for topic is "Is WordPress or CMS is beneficial for developers, and is the web development of certain field getting ended? Why do developers dont like such such work"
Idk if this is a right topic to discuss on but i just wanted to suggest this topic to atleast know everyone's views on Content management systems like WordPress, wix etc.

**Reply by kaiwalyakoparkar** [reply]
Good topic but do you think beginners would be able to speak on this topic?

**Reply by ItsMeIshan** [reply]
> Good topic but do you think beginners would be able to speak on this topic?

Yea i think only a few or may be more folks can speak but not all will be able to speak on this i suggested this because i wanna have a discussion with you all folks! Surely we can go with other topics too!

**Reply by ashwinexe** [reply]
> Good topic but do you think beginners would be able to speak on this topic?

I think beginners would rather like to listen to this as this is a new trend and not many understand why to use (or not use) low code applications. Eddie also made a video on why he used low code for his own website and has explained some very good points,
Take a look at it here -> https://www.youtube.com/watch?v=rIZrAaOL2LA

**Reply by ItsMeIshan** [reply]
> > Good topic but do you think beginners would be able to speak on this topic?
> 
> I think beginners would rather like to listen to this as this is a new trend and not many understand why to use (or not use) low code applications. Eddie also made a video on why he used low code for his own website and has explained some very good points,
> Take a look at it here -> https://www.youtube.com/watch?v=rIZrAaOL2LA

Yes @ashwinexe i have seen this and this video only made me to think more about this topic and a discussion with you all! But your point is right that beginners wouldn't speak much on this.

**Reply by naomi-lgbt** [reply]
I think, given that the purpose of these calls is to give our members an opportunity to practise their speaking skills, selecting a topic that is not inclusive for everyone would be counterintuitive.

**Comment by verma-kunal** [comment]
This is a nice idea by **Shivam**! Do share your suggestions 😇
![Screenshot 2021-06-18 at 8 22 55 PM](https://user-images.githubusercontent.com/72245772/122579868-f8907d00-d072-11eb-85a3-308a41f4e3ab.png)

**Reply by ItsMeIshan** [reply]
Yes i think this is awesome and would make more and more folks speak up💥💥

**Comment by verma-kunal** [comment]
Topic suggested by @kaiwalyakoparkar 

![Screenshot 2021-06-19 at 12 26 49 AM](https://user-images.githubusercontent.com/72245772/122605418-0bb44480-d095-11eb-8fb3-c26f34e1bf5b.png)

**Reply by verma-kunal** [reply]
I think this is a nice topic & we can all analyse that where things went wrong or did not go as expected and try to improve upon them in the coming time !!

**Reply by ashwinexe** [reply]
I like this topic, we always see everyone's tip of the icebergs in social media (i.e success stories) Its great to know what all failures we all have went, which made us strong to this moment. Looking forward to speak and listen on it :)

**Reply by verma-kunal** [reply]
True !!

**Reply by ItsMeIshan** [reply]
This topic is really awesome we can definitely consider it for speaking as alot of folks including me too can learn alot and speak up on the stage ✨

**Reply by viki4545** [reply]
Yeah @verma-kunal  its really a awesome topic 👍 suggested by @kaiwalyakoparkar

**Reply by kaiwalyakoparkar** [reply]
Should we select this?

**Comment by kaiwalyakoparkar** [comment]
:heart:  Saturday Public Speaking session **19/ 06/ 2021 6:00 PM IST** :heart:

:point_down: Feel free to add your suggestions for improvement of this initiative and idea for next session :point_down:

Also share about your experience on socials (And don't forget to tag your peers :wink: ) and how it is helping you!

**Reply by verma-kunal** [reply]
Topic: **Tell us about your failures from this week**

**Comment by verma-kunal** [comment]
Hey folks 👋  
Would love to have your suggestions on the topic "**Hackathon Memories**" for the public speaking session !!

**Reply by kaiwalyakoparkar** [reply]
Hehe! Awesome topic!

**Comment by ghost** [comment]
Next Public Speaking Session Suggestion: **What is your dream project you want to build?**

Examples: Let's say someone name Rahul wants to build the next chromium

**Reply by Akshu-on-github** [reply]
Wolf Gupta, is that you?
Jokes apart, that sounds pretty interesting 🤔 We could also talk about the ideas we wrote off because they weren't feasible. It has the potential to be a good rubber duck session 👍

**Reply by ghost** [reply]
> Wolf Gupta, is that you?
> Jokes apart, that sounds pretty interesting 🤔 We could also talk about the ideas we wrote off because they weren't feasible. It has the potential to be a good rubber duck session 👍

I didn't understand the wolf thing 😓?

**Reply by kaiwalyakoparkar** [reply]
@krishdevdb That was a joke. "Wolf Gupta" is a character created by Whitehat junior which according to them got a job at Google at the age of 13 and is making crores. WHJ accepted that this was imaginary and was intended to attract more audiences to their portal. Nothing serious :smile:

**Reply by ghost** [reply]
> "Wolf Gupta" is a character created by Whitehat junior

Oh 😮 I thought that there is someone named Wolf Gupta in our community and Akshu misunderstood me for him. Sorry 😓

**Comment by ashwinexe** [comment]
Saturday Public Speaking session 07/ 08/ 2021 6:00 PM IST 

My suggestion for this week 👉🏻 **How much we could improve if we simply come out of our comfort zone**
(We can discuss things we wanted to do, but we simply can't because of possible imposter syndrome or its just outside of our comfort, and what should we do in order to step out of comfort zone for the good practices)
Let us know what you'd like to speak about in the comments 😄 

If you gain some good knowledge from the session, do share what you learnt, and don't forget to tag us ;)
